client/lib/purchases/index.js

```diff
@@ -308,6 +308,10 @@ function hasCreditCardData( purchase ) {
 	return Boolean( purchase.payment.creditCard.expiryMoment );
 }
 
+function shouldAddPaymentSourceInsteadOfRenewingNow( expiryMoment ) {
+	return expiryMoment > moment().add( 90, 'days' );
```

Comment:
I wonder if this (and the one other place in the existing code that does 90 days) should be checking for "3 months" instead?

I noticed that that's what the server-side code does, and although 3 months and 90 days are almost the exact same thing, they can be off a couple days a year :)

(Or alternatively, we could change the server-side code to be 90 days... it probably doesn't matter.)

Reply:
Done!

Reply:
Done